### ISSUE TITLE
PlayerSheetV2 | Maintain Scroll Position

### ISSUE THREAD

**Issue by Oliver-Akins** [Oliver-Akins/foundry.dungeon#131]
The scroll position should be maintained. As it's annoying AF that it jumps around when rerendering (like in the Inventory item expanding/collapsing)
